service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/CreateGcsBucketStep.java

```diff
@@ -56,6 +56,10 @@ public StepResult doStep(FlightContext flightContext)
             .map(GcsApiConversions::toGcsApiRulesList)
             .orElse(Collections.emptyList()));
 
+    BucketInfo.IamConfiguration iamConfiguration =
```

Comment:
I wonder if we will see an application that wants to use object-level IAM.
If that comes up, we can expose the parameter.

Reply:
If we did that, I think we would be trusting applications to manage their own IAM and they could e.g. share data with anyone. Maybe that's actually OK for trusted/reviewed applications, but there would probably be some compliance work required.

Either way, something to think about when applications are better defined